### DISCUSSION TITLE
Charging even though it's above the charge limit.

### DISCUSSION THREAD

**Discussion by calvinvadivelu** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#744] [Q&A]
I can't tell if I'm using the application wrong or maybe my settings are incorrect. It seems that the battery keeps charging even though it's above the charge limit. I've set the limit to 80 and will only discharge when I click the "Discharge" button. I had thought that the app would limit it without me having to go in and click discharge.

![Screen Shot 2022-08-01 at 8 44 08 AM](https://user-images.githubusercontent.com/13733179/182151089-28a3c56d-315f-47c5-8705-ef72f659217a.png)

![Screen Shot 2022-08-01 at 8 47 44 AM](https://user-images.githubusercontent.com/13733179/182151168-6bfa37be-5a59-4f85-9379-b67a298df8b8.png)

**Comment by MatthiasKerbl** [comment]
Hi @calvinvadivelu,

Which MacBook model do you have?
Which macOS version do you have?
Are you familiar with these feature explanations?
https://apphousekitchen.com/feature-explanation-stop-charging-when-powered-off-app-closed/
https://apphousekitchen.com/feature-explanation-stop-charging-when-sleeping/
https://apphousekitchen.com/feature-explanation-disable-sleep-until-charge-limit/

Cheers,
Matthias
